### DISCUSSION TITLE
Open Science Plans

### DISCUSSION THREAD

**Discussion by fridlind** [ARM-Development/comble-mip#68] [Announcements]
Thanks to everyone for participating in COMBLE-MIP, and thank you for your patience while we learn on the job at the new DOE ARM JupyterHub Workbench!

The first phase of our JupyterHub environment deployment has been focused on sharing our scripts and testing environment. Thank you for your questions and GitHub commits. We think this has been a success thus far.

The next phase of our JupyterHub environment deployment is intended to move "sandbox" results to "staged" directories, which are intended to indicate "ready-to-submit" model output files (after addressing any accidental omissions, and sign or unit errors). The staged files will ultimately be committed to the DOE ARM repository as user supplied public data sets, along with 2D and 3D outputs. Each participant will commit their own files, and we'll provide instructions on how to do that. These committed data sets will be provided with DOIs that will be used in the forthcoming manuscripts.

What about open science at this point? As we accumulate "staged" results, we plan to automate plotting scripts that will be updated on a rolling basis to show all the staged model results, identified by model name as submitted. This is common within intercomparison groups to show models by name, but perhaps somewhat new to put the results on a public web site during project progression. If anyone has any concerns about this, please contact any one of us.

— Ann Fridlind, Tim Juliano, and Florian Tornow

**Comment by anddebol** [comment]
Sorry for the late response, should directory be  `les_output/<model_name>/staged/` ?

**Comment by twjuliano** [comment]
> Sorry for the late response, should directory be `les_output/<model_name>/staged/` ?

@anddebol yes, that is correct!
